packages/create-sitecore-jss/src/templates/nextjs/src/lib/page-props-factory/plugins/project.ts

```diff
@@ -0,0 +1,25 @@
+import { SitecorePageProps } from 'lib/page-props';
+import chalk from 'chalk';
+import fs from 'fs';
+import { Plugin } from '..';
+
+class ProjectPlugin implements Plugin {
+  order = 1;
+
+  async exec(props: SitecorePageProps) {
+    const isProjectImplemented =
+      props.site.project && fs.existsSync(`${process.cwd()}/src/projects/${props.site.project}`);
```

Comment:
Don't we need a short-circuit here if `props.site.project` is undefined?
```suggestion
    if (!props.site.project) {
      // Not using a project
      return props;
    }
    const isProjectImplemented = fs.existsSync(`${process.cwd()}/src/projects/${props.site.project}`);
```

Currently, it looks as though the warning will be displayed even if their site doesn't use a project.

Reply:
Good call! Agree